C2.md

```diff
@@ -271,3 +284,340 @@ in minifi.properties to activate the file update trigger specify
+    },
+    "queues": {
+        "8368e3c8-015a-1003-52ca-83af40ec1332": {
+            "dataSize": 40,
+            "dataSizeMax": 1048576,
+            "name": "GenerateFlowFile/success/LogAttribute",
+            "size": 4,
+            "sizeMax": 0,
+            "uuid": "8368e3c8-015a-1003-52ca-83af40ec1332"
+        }
+    },
+    "processorStatuses": [
+        {
+            "id": "5128e3c8-015a-1000-79ca-83af40ec1990",
+            "groupId": "2438e3c8-015a-1000-79ca-83af40ec1990",
+            "bytesRead": 0,
+            "bytesWritten": 0,
+            "flowFilesIn": 0,
+            "flowFilesOut": 0,
+            "bytesIn": 0,
+            "bytesOut": 0,
+            "invocations": 0,
+            "processingNanos": 0,
+            "activeThreadCount": -1,
+            "terminatedThreadCount": -1,
+            "running": true
+        },
+        {
+            "id": "4fe2d51d-076a-49b0-88de-5cf5adf52b8f",
+            "groupId": "2438e3c8-015a-1000-79ca-83af40ec1990",
+            "bytesRead": 0,
+            "bytesWritten": 40,
+            "flowFilesIn": 0,
+            "flowFilesOut": 4,
+            "bytesIn": 0,
+            "bytesOut": 40,
+            "invocations": 4,
+            "processingNanos": 2119148,
+            "activeThreadCount": -1,
+            "terminatedThreadCount": -1,
+            "running": true
+        }
+    ],
+    "flowId": "96273342-b9fe-11ef-a0ad-10f60a596f64"
+}
+```
+
+### QueueMetrics
+
+Contains information about the queues in the flow, including the contained data and number of flow files.
+
+```
+"QueueMetrics": {
+    "GenerateFlowFile/success/LogAttribute": {
+        "datasize": "40",
+        "datasizemax": "1048576",
+        "queued": "4",
+        "queuedmax": "0"
+    }
+}
+```
+
+### RepositoryMetrics
+
+Contains information about the repositories in the agent, including the number of entries, size, and whether the repository is full.
+
+
+```
+"RepositoryMetrics": {
+    "repo_name": {
+        "entryCount": 0,
+        "full": false,
+        "maxSize": 0,
+        "running": true,
+        "size": 0
+    },
+    "ff": {
+        "entryCount": 0,
+        "full": false,
+        "maxSize": 0,
+        "running": true,
+        "size": 0
+    },
+    "org::apache::nifi::minifi::core::repository::VolatileContentRepository": {
+        "entryCount": 4,
+        "full": false,
+        "maxSize": 7864320,
+        "running": true,
+        "size": 40
+    }
+}
+```
+
+### Processor Metric Response Nodes
+
+Each processor can have its own metrics. These metric nodes can be configured in the minifi.properties by requesting metrics in the <ProcessorTye>Metric format, for example GetTCPMetrics to request metrics for the GetTCP processors. Besides configuring processor metrics directly, they can also be configured using regular expressions with the `processorMetrics/` prefix. For example `processorMetrics/Get.*Metrics` will match all processor metrics that start with Get.
```

Comment:
```suggestion
Each processor can have its own metrics. These metric nodes can be configured in the minifi.properties by requesting metrics in the \<ProcessorType\>Metric format, for example GetTCPMetrics to request metrics for the GetTCP processors. Besides configuring processor metrics directly, they can also be configured using regular expressions with the `processorMetrics/` prefix. For example `processorMetrics/Get.*Metrics` will match all processor metrics that start with Get.
```

Reply:
Fixed in https://github.com/apache/nifi-minifi-cpp/pull/1909/commits/2723bace043147136941c68f86216fcd61d74a36

Reply:
the typo is still not fixed in your update @lordgamez Tye vs Type

Reply:
Thanks, I didn't see the type only fixated on the missing backslashes, fixed in https://github.com/apache/nifi-minifi-cpp/pull/1909/commits/24c93329eec9e7a505d318804ba9b78589d5c9d3